src/org/labkey/test/util/EscapeUtil.java

```diff
@@ -149,4 +149,19 @@ public static String getTextChoiceValidatorExpression(List<String> options)
                 .map(value -> value.replaceAll("([\\\\|])", "\\\\$1"))
                 .collect(Collectors.joining("|"));
     }
+
+    public static String getSqlQuotedValue(String value)
+    {
+        return String.format("\"%s\"", value.replaceAll("\"", "\"\""));
+    }
+
+    public static String getMarkupEscapedValue(String value)
+    {
+        return value
+                .replace("&", "&amp;")
+                .replace("<", "&lt;")
+                .replace(">", "&gt;")
+                .replace("\"", "&quot;")
+                .replace("'", "&apos;");
+    }
```

Comment:
The Apache commons text library provides this functionality. No need to reinvent. If you must define a method, just farm out the work to the established library.
```suggestion
    public static String getMarkupEscapedValue(String value)
    {
        return StringEscapeUtils.escapeXml11(value);
    }
```

Reply:
Thanks!